src/lib/Theme.ts

```diff
@@ -1006,9 +1006,11 @@ export default class Theme {
                 path.join(Theme.BUILD_FOLDER, `${assetHash}_themeBundle.common.js`),
                 'utf-8'
             );
-
+            let customTemplates=[]
             const themeBundle = evaluateModule(bundleFiles);
-            const customTemplates = themeBundle.getCustomTemplates();
+            if(themeBundle && themeBundle.getCustomTemplates()){
```

Comment:
if `getCustomTemplates` is not defined inside the theme it will break again @ekata123

Reply:
@vivek8690 resolved comment